src/wasm-lib/kcl/src/std/shapes.rs

```diff
@@ -145,6 +148,67 @@ async fn inner_circle(
     Ok(new_sketch)
 }
 
+/// Data for drawing a 3-point circle
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, ts_rs::TS, JsonSchema)]
+#[ts(export)]
+#[serde(rename_all = "camelCase")]
+pub struct CircleThreePointData {
+    /// Point one for circle derivation.
+    pub p1: [f64; 2],
+    /// Point two for circle derivation.
+    pub p2: [f64; 2],
+    /// Point three for circle derivation.
+    pub p3: [f64; 2],
```

Comment:
I had wanted to do `[[f64;2]; 3]` since they are one unit, and are order independent, but this would be the only case of this so far, so @jtran suggested either this as a keyword arg or individual arguments. The former I believe is the most appropriate because it should be seen as one data unit.

Reply:
What you currently have is a single positional param that's an object. See `loft.rs` in #4757. The key thing is:

```rust
#[stdlib {
    name = "loft",
    keywords = true,
```

...which opts-in to using keyword args.